SignalServiceKit.podspec

```diff
@@ -37,7 +37,7 @@ An Objective-C library for communicating with the Signal messaging service.
   s.dependency 'AFNetworking'
   s.dependency 'AxolotlKit'
   s.dependency 'Mantle'
-  s.dependency 'YapDatabase/SQLCipher'
+  s.dependency 'YapDatabase/SQLCipher', '~> 2.9.3'
```

Comment:
LGTM, want to include the podfile.lock that you get after running pod install? It keeps future PR's more concise.